src/main.ts

```diff
@@ -55,20 +55,7 @@ async function bootstrap() {
   }
 
   // Setup swagger. Todo: do we want this in production and staging?
-  const config = new DocumentBuilder()
-    .setTitle('gipfeli.io')
-    .setDescription('API Documentation for the gipfeli.io backend API.')
-    .setVersion('1.0')
-    .addBearerAuth()
-    .build();
-  const document = SwaggerModule.createDocument(app, config);
-  SwaggerModule.setup('api', app, document, {
-    customSiteTitle: 'gipfeli.io API Documentation',
-    swaggerOptions: {
-      operationsSorter: 'alpha',
-      tagsSorter: 'alpha',
-    },
-  });
+  swaggerSetup(app);
```

Comment:
I suggest to add the swagger documentation to staging and production but remove the feature where you can test the api. If this can somehow be split?

Reply:
Fixed in https://github.com/gipfeli-io/gipfeli-api/pull/103 (ugly, but seems to be the way :))